src/util/error/code.js

```diff
@@ -66,7 +66,12 @@ const Code: CodeType = {
   /**
    * The provider doesn't implement the called api
    */
-  METHOD_NOT_IMPLEMENTED: 3002
+  METHOD_NOT_IMPLEMENTED: 3002,
+
+  /**
+   * The server responded with status import or pre convert
+   */
+  MEDIA_STATUS_NOT_READY: 3003
```

Comment:
should be 2xxx error, move to SERVICE address range

Reply:
@RoyBregman ?

Reply:
fixed and pushed